src/domain/synthetics/markets/useMarketsInfoRequest.ts

```diff
@@ -519,11 +531,11 @@ export function useMarketsInfoRequest(chainId: number): MarketsInfoResult {
           ),
 
           claimableFundingAmountLong: dataStoreValues.claimableFundingAmountLong
-            ? BigNumber.from(dataStoreValues.claimableFundingAmountLong?.returnValues[0])
+            ? BigNumber.from(dataStoreValues.claimableFundingAmountLong?.returnValues[0]).div(marketDivisor)
             : undefined,
 
           claimableFundingAmountShort: dataStoreValues.claimableFundingAmountShort
-            ? BigNumber.from(dataStoreValues.claimableFundingAmountShort?.returnValues[0])
+            ? BigNumber.from(dataStoreValues.claimableFundingAmountShort?.returnValues[0]).div(marketDivisor)
```

Comment:
if i'm checking correctly contracts use this value as is https://github.com/gmx-io/gmx-synthetics/blob/updates/contracts/market/MarketUtils.sol#L604, no need to use divisor

Reply:
discussed in chat that we need to divide here